x-pack/plugins/ml/public/application/model_management/deployment_setup.tsx

```diff
@@ -63,300 +129,496 @@ export const DeploymentSetup: FC<DeploymentSetupProps> = ({
+                            ),
+                          }),
+                      optimized: v.value,
+                    });
+                  }}
+                  data-test-subj={`mlModelsStartDeploymentModalOptimized_${v.value}`}
+                />
+                <EuiSpacer size="m" />
+              </Fragment>
+            );
+          })}
+        </EuiFormFieldset>
+      </EuiFormRow>
+
+      <EuiAccordion
+        id={'modelDeploymentAdvancedSettings'}
+        buttonContent={
+          <FormattedMessage
+            id="xpack.ml.trainedModels.modelsList.startDeployment.advancedSettingsLabel"
+            defaultMessage="Advanced configurations"
+          />
+        }
+        initialIsOpen={isUpdate}
+        data-test-subj={'mlModelsStartDeploymentModalAdvancedConfiguration'}
+      >
+        <EuiSpacer size={'m'} />
+
+        <EuiPanel hasBorder hasShadow={false}>
           <EuiFormRow
+            hasChildLabel={false}
+            fullWidth
             label={
-              <FormattedMessage
-                id="xpack.ml.trainedModels.modelsList.startDeployment.threadsPerAllocationLabel"
-                defaultMessage="Threads per allocation"
-              />
+              showNodeInfo ? (
+                <FormattedMessage
+                  id="xpack.ml.trainedModels.modelsList.startDeployment.vCpuUsageLabel"
+                  defaultMessage="vCPUs usage level"
+                />
+              ) : (
+                <FormattedMessage
+                  id="xpack.ml.trainedModels.modelsList.startDeployment.vCUUsageLabel"
+                  defaultMessage="VCU usage level"
+                />
+              )
             }
-            hasChildLabel={false}
-            isDisabled={disableThreadingControls}
           >
-            <EuiButtonGroup
-              isDisabled={disableThreadingControls}
-              legend={i18n.translate(
-                'xpack.ml.trainedModels.modelsList.startDeployment.threadsPerAllocationLegend',
-                {
-                  defaultMessage: 'Threads per allocation selector',
-                }
-              )}
-              name={'threadsPerAllocation'}
-              isFullWidth
-              idSelected={
-                disableThreadingControls
-                  ? '1'
-                  : threadsPerAllocationsOptions.find((v) => v.value === threadsPerAllocations)!.id
-              }
-              onChange={(optionId) => {
-                const value = threadsPerAllocationsOptions.find((v) => v.id === optionId)!.value;
-                onConfigChange({ ...config, threadsPerAllocations: value });
+            <EuiRange
+              fullWidth
+              id={'vCpuLevel'}
+              min={0.5}
+              max={2.5}
+              step={1}
+              tickInterval={1}
+              value={vCpuLevelMap[config.vCPUUsage].value}
+              onChange={(e) => {
+                const result = Object.entries(vCpuLevelMap).find(
```

Comment:
`Object.values` could be used here

Reply:
I actually need entires, because at like 539 I use the key from the `result`